projects/firmware_103/src/main.c

```diff
@@ -0,0 +1,74 @@
+#include "../../../libraries/ms-common/inc/adc.h"
+#include "delay.h"
+#include "gpio.h"
+#include "gpio_it.h"
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+
+typedef enum { B2_BUTTON = 0, NUM_BUTTONS } Button;

```

Comment:
Add a comma after NUM_BUTTONS, so when you run `make format`, it will format itself correctly

Reply:
Also this may be a bit overkill for an enum, we can just use `#define` in this case